src/tests/e2e/emails/emails.spec.ts

```diff
@@ -0,0 +1,467 @@
+import { expect, test } from "@playwright/test";
+import path from "node:path";
+
+import { EVENT_DATA, TemplatesPage } from "./utils";
+import type { Editor } from "./utils";
+
+test.describe.configure({ mode: "serial" });
```

Comment:
oj oj, redflag - testy powinny móc bez problemu lecieć in parallel - jak używasz serial to znaczy, że coś robisz źle i testy są zależne od siebie